lib/active_interaction/backports.rb

```diff
@@ -4,18 +4,6 @@
 require 'active_support/core_ext'
 
 module ActiveInteraction
-  class GroupedInput # rubocop:disable Style/Documentation
-    # Required for Ruby <= 1.9.3.
-    def [](name)
-      send(name)
-    end unless method_defined?(:[])
-
-    # Required for Ruby <= 1.9.3.
-    def []=(name, value)
-      send("#{name}=", value)
-    end unless method_defined?(:[]=)
-  end
-
```

Comment:
I planned on leaving this in. When I changed the gem spec to require Ruby `>= 2`, I realized that this served absolutely no purpose. If we wanted to continue to unofficially support Ruby 1.9.3, we'd need to keep Ruby `>= 1.9.3` in the gem spec. And that basically means officially supporting Ruby 1.9.3, which defeats the whole point.

All that to say, I've changed my mind. I'm fine with dropping support for 1.9.3. If someone wants to use ActiveInteraction on 1.9.3, they need to use some version < 3.